x-pack/platform/plugins/shared/alerting/server/alert_deletion/lib/test_utils.ts

```diff
@@ -7,87 +7,145 @@
 
 export const inactiveAlertsQuery = (days: number = 30, spaceId: string = 'space-1') => ({
```

Comment:
Since these seem to be used just in tests, I wonder if they can be coded as KQL and then expand to DSL from there.

Reply:
I actually find it helpful to see the expanded DSL during tests, especially for the inactive query which has a lot of parentheses in the KQL.